main/settings.py

```diff
@@ -28,9 +29,9 @@
 SECRET_KEY = "7ibm!g0j@gs7kszwav!6$*lar(+!!l3tpm@09s%csl2bj)l+p4"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```

Comment:
For deployment take this DEBUG value from environment variables else for development make this True.